Automattic/jetpack
# Tiled Gallery: Fix height of add item for compat with FSE

Related to: 256-gh-Automattic/view-design

This change updates the CSS for the editor view of the Tiled Gallery block to create consistency between the post editor and the new FSE site editor. In the new site editor, the existing CSS that adds `height: 100%` to the add item button unintentionally causes the button to be much taller visually than expected. This appears to be due to the different markup used in the site editor, when the block is rendered within a template part in the editor canvas.

I couldn't find any issues with removing this CSS rule (as the button has a minimum height of 100px anyway), but please let me know if you can think of a reason it needs to stay in.

#### Changes proposed in this Pull Request:

* Remove unneeded `height: 100%` from the add item button in the Tiled Gallery block's editor UI

#### Screenshots

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/113985818-e5517c00-988f-11eb-939b-0b39ca4d7218.png) | ![image](https://user-images.githubusercontent.com/14988353/113985786-dd91d780-988f-11eb-8d95-83befe8da3e2.png) |

#### Jetpack product discussion


256-gh-Automattic/view-design

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply this change using an FSE-compatible theme and from the site editor (e.g. `/wp-admin/admin.php?page=gutenberg-edit-site`) test the Tiled Gallery block
* From the standard post editor, compare and ensure the editor UI looks basically the same
* Try with a couple of other themes and non-FSE themes to ensure this doesn't introduce regressions / degrade the editing experience
